app/views/tracks/concepts/show/joined.html.haml

```diff
@@ -1,2 +1,13 @@
 .md-container
   %h2 #{@track.title}: #{@concept.name}
+
+  - if @solution
+    - if @solution.completed?
+      Exercise Completed
+    - else
+      Exercise Started
+      = link_to "Continue", edit_solution_path(@solution.uuid)
+      = link_to "Complete", complete_track_concept_path(@track, @concept), method: :patch
+  - else
+    Exercise not started
+    = button_to "Start exercise", start_track_concept_url(@track, @concept), method: :patch
```

Comment:
This is all just scaffolding for enabling us to walk through everything. It'll all get done properly once the designs and URLs are finalised.